src/frontend-samples/tree-samples/drag-and-drop-tree/DragAndDropTree.tsx

```diff
@@ -0,0 +1,260 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file should be named DragAndDropTreeApp.tsx to match the convention of all the other samples.